softwerkskammer/lib/activities/views/index.pug

```diff
@@ -8,18 +9,22 @@ block content
   .row
     .col-md-12
       .page-header
-        .btn-group.pull-right
-          .btn-group
-            a.btn.btn-default.dropdown-toggle(data-toggle='dropdown')
-              | #{range} &nbsp;
-              span.caret
-            ul.dropdown-menu
-              li: a(href='/activities/upcoming') #{t('activities.upcoming')}
-              li: a(href='/activities/past') #{t('activities.past')}
-              li: a(href='/activities') #{t('general.all')}
-              li: a(href='/activities/gdcr') #{t('general.gdcr_activities')}
-          a.btn.btn-default(href=webcalURL, title=t('activities.export_subscribe')): i.fa.fa-calendar.fa-fw
-          if (accessrights.canCreateActivity())
-            a.btn.btn-default(href='new/', title=t('activities.new')): i.fa.fa-file-o.fa-fw
+        form(role='form', method='POST', action='clone-from-meetup/')
```

Comment:
Warum das Form um alles herum? - Ich fände es besser, wenn es nur um den Meetup button ist

Reply:
Hab ich probiert, aber dann wird die Buttonleiste unterbrochen (also in zwei Stücke gehackt). Wenn Du das reparieren kannst, ändere ich das gern wieder.

Reply:
OK